GitCommands/ArgumentBuilderExtensions.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel;
+using GitCommands.Git;
+using JetBrains.Annotations;
+
+namespace GitCommands
+{
+    /// <summary>
+    /// Extension methods for working with <see cref="ArgumentBuilder"/>.
+    /// </summary>
+    public static class ArgumentBuilderExtensions
+    {
+        /// <summary>
+        /// Adds <paramref name="ifConditionTrue"/> to the argument list if <paramref name="condition"/>
+        /// is <c>true</c>.
```

Comment:
:bulb: Most of the places where `true`, `false`, and `null` are used in documentation could benefit from the `<see langword="true"/>` syntax. This is supported directly by recent versions of Visual Studio, so if you just type `true` and press <kbd>Ctrl</kbd>+<kbd>Space</kbd> it will expand to the correct form.

Reply:
I've seen this in the past but wondered about the benefit. It's semantically clearer to a computer, but not necessarily to a human. Are there tools that benefit from this?

Reply:
1. It's much easier to type (code completion)
2. QuickInfo in Visual Studio uses it for highlighting
3. Sandcastle Help File Builder uses it if you produce documentation output
4. It would be the basis for keyword handling in other tools as well

Reply:
I agree if you're comparing with a plain keyword, but `<c>null</c>` is going to be formatted the same, and enjoys the same completion, at least with my setup :)

Reply:
`<c>null</c>` will be rendered as monospace, but it won't be converted for other languages the same way `<see langword="null" />` will - `null` is a c# keyword, for vs it will get rendered as `Nothing`